diff -r cb5cf75697e1 Makefile.conf
--- a/Makefile.conf	Thu Aug 01 07:39:45 2013 +0000
+++ b/Makefile.conf	Mon Sep 15 14:26:19 2014 +0000
@@ -33,7 +33,7 @@
 # directory for executable
 BINDIR=/usr/bin
 # directory for perl modules
-PERLDIR=/usr/share/perl5
+PERLDIR=/usr/local/lib64/perl5
 # directory for some AMC executables, not to be used by end-user
 MODSDIR=/usr/lib/AMC
 # directory for LaTeX style file
@@ -73,7 +73,7 @@
 
 # BUILD : options to compile with netpbm / opencv libraries
 
-GCC_NETPBM=-lnetpbm
+GCC_NETPBM=-I/usr/include/netpbm/ -lnetpbm
 
 GCC_OPENCV=-I /usr/include/opencv
 GCC_OPENCV_LIBS=auto
diff -r cb5cf75697e1 doc/Makefile
--- a/doc/Makefile	Thu Aug 01 07:39:45 2013 +0000
+++ b/doc/Makefile	Mon Sep 15 14:26:19 2014 +0000
@@ -37,6 +37,7 @@
 	rm -f modeles/**/*.tgz
 	rm -f html/auto-multiple-choice.**/*.html
 	rm -f html/images/callouts/*.png html/images/*.png
+	rm -f html/index.html 
 	rm -f *~
 
 html/images/callouts/%.png: img_src/callouts/%.svg
diff -r cb5cf75697e1 doc/sty/Makefile
--- a/doc/sty/Makefile	Thu Aug 01 07:39:45 2013 +0000
+++ b/doc/sty/Makefile	Mon Sep 15 14:26:19 2014 +0000
@@ -22,6 +22,9 @@
 PACK=automultiplechoice
 
 all: FORCE
+	sed -i s?'@/PACKAGE_V_DEB/@'?1.2.1-gentoo?g $(PACK).dtx
+	sed -i s?'@/PACKAGE_V_SVN/@'?1.2.1-hg?g $(PACK).dtx
+	grep '1\.2\.1' $(PACK).dtx
 	pdflatex $(PACK).dtx
 	pdflatex sample-amc.tex
 	pdflatex sample-separate.tex
